### DISCUSSION TITLE
Checking if and what external display is connected

### DISCUSSION THREAD

**Discussion by stefanhoefer** [ttscoff/bunch-docs#250] [Q&A] [answered]
Hello!

Is there a way to check if and what external display(s) is/are connected? I would like to use a different Moom window arrangement depending on my screen setup.

Thanks for any help!

**Comment by stefanhoefer** [comment, marked as answer]
For anyone also interested in this: I got it working by using this command: $ system_profiler SPDisplaysDataType | grep -c Resolution . I stored the result in a variable and used conditional logic to determine which window set Moom should use.

**Comment by ttscoff** [comment]
Great tip, thanks for sharing!
